src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -89,6 +89,34 @@ public AssemblyScannerResults GetScannableAssemblies()
                 }
             }
 
+            var platformAssembliesString = (string)AppDomain.CurrentDomain.GetData("TRUSTED_PLATFORM_ASSEMBLIES");
+
+            if (platformAssembliesString != null)
+            {
+                var platformAssemblies = platformAssembliesString.Split(Path.PathSeparator);
+
+                foreach (var platformAssembly in platformAssemblies)
+                {
+                    try
+                    {
+                        var token = AssemblyName.GetAssemblyName(platformAssembly).GetPublicKeyToken();
+                        var tokenString = BitConverter.ToString(token).Replace("-", string.Empty).ToLowerInvariant();
+
+                        Console.WriteLine($"{platformAssembly} {tokenString}");
```

Comment:
Didn't see the Console.WriteLines :)

Reply:
Those are from the debugging commits I added because of https://github.com/Particular/NServiceBus/pull/5081#issuecomment-346514972

I suppose I should mark this WIP for now until I get an update on the issue I filed.